test/Microsoft.DotNet.Cli.Utils.Tests/StreamForwarderTests.cs

```diff
@@ -39,16 +38,18 @@ public static IEnumerable<object[]> ForwardingTheoryVariations
                 return new[]
                 {
                     new object[] { "123", new string[]{"123"} },
-                    new object[] { "123\n", new string[] {"123"} },
-                    new object[] { "123\r\n", new string[] {"123"} },
+                    new object[] { "123\n", new string[] {"123", ""} },
```

Comment:
I do, but I also think that `Console.Write(…)` and `Console.WriteLine(…)` should produce different output. Which is why I now think that #2244 is a better solution, it doesn't have this problem.